### DISCUSSION TITLE
Does Ice support RDMA(RemoteDirect Memory Access)?

### DISCUSSION THREAD

**Discussion by cqdjsp** [zeroc-ice/ice#1484] [Q&A] [answered]
My transmission speed requirement is 60Gb/s, but I found that Ice's CPU usage rate is too high under the default configuration, resulting in the transmission speed not meeting the requirements. Therefore, I would like to know if Ice supports RDMA？

**Comment by bernardnormier** [comment, marked as answer]
Ice does not support RDMA.

The reliable transports supported by Ice (TCP, WebSocket, Bluetooth RFCOMM...) are all "single data pipe" transports like TCP.

**Reply by cqdjsp** [reply]
Understood. Thank you for your answer.
